app/views/moderations/_mod_sidebar_right.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="crayons-card__body mod-saying-hello">
       <p class="fw-medium fs-2xl">Hello! 👋</p>
       <p>
-        Thank you for helping to keep <%= ApplicationConfig["COMMUNITY_NAME"] %> safe! ❤️
+        Thank you for helping to keep <%= SiteConfig.community_name %> safe! ❤️
```

Comment:
**Comment (not-blocking):**This probably should have been using the method name community_name from the application helper. We probably missed it in the first pass.